src/integer_mod_q/polynomial_ring_zq/get.rs

```diff
@@ -26,7 +74,6 @@ impl PolynomialRingZq {
     /// let poly_ring = PolynomialRingZq::from((&poly, &modulus));
     ///
     /// let poly_ring_mod = poly_ring.get_mod();
-    ///
     /// assert_eq!(modulus, poly_ring_mod);
```

Comment:
Why did you delete this line?
It separates the function to be tested from the assert logic.